release-notes.txt

```diff
@@ -36,7 +36,7 @@ The 2021.01 release includes:
 In order to improve community interaction, during this release cycle there was
 some work to migrate from a mailing list based communication to forum-based
 communication. We are proud to present the RIOT-OS forum available at
-https://forum.riot-os.org
+forum.riot-os.org
```

Comment:
Why remove the `https://`?

Reply:
just to be consistent with the other links (e.g irc.freenode.net, etc)

Reply:
is it ok?